Exercise_1.py

```diff
@@ -1,18 +1,26 @@
 class myStack:
      def __init__(self):
+          self.res  = []
          
      def isEmpty(self):
+          if len(self.res):
+               return True
          
      def push(self, item):
+          self.res.append(item)
          
      def pop(self):
-        
+        self.res.pop()
```

Comment:
Please add edge case to prevent from operating on a empty stack.

Reply:
Done!